user_signup/app.yaml

```diff
@@ -0,0 +1,11 @@
+runtime: python27
+api_version: 1
+threadsafe: true
+
+handlers:
+- url: /.*
+  script: main.app
+
+libraries:
+- name: jinja2
+  version: latest
```

Comment:
Generally preferred to include newlines at the end of files.  Not a big deal though.